Gemfile

```diff
@@ -76,7 +76,7 @@ gem 'stringex', git: 'https://github.com/pulibrary/stringex.git', tag: 'vpton.2.
 gem 'mail_form'
 gem 'string_rtl'
 
-gem 'requests', git: 'https://github.com/pulibrary/requests.git'
+gem 'requests', git: 'https://github.com/pulibrary/requests.git', branch: 'i801-hathi-oclc'
 
 gem 'borrow_direct', git: 'https://github.com/pulibrary/borrow_direct.git', branch: 'generate_query_encoding_fix'
```

Comment:
This PR is using the branch `i801-hathi-oclc` from the requests gem. First test and then merge the `requests` Pr https://github.com/pulibrary/orangelight/tree/2230-hathi-access-oclc-lookup to master. Then update the requests version on this PR.